src/app/hooks/auth/useAuthInterceptor.ts

```
@@ -6,49 +6,61 @@ import { refreshAuthToken } from "@/app/api/auth/[...nextauth]/auth";
-      if (!session.user.refreshToken) return;
-
+    const refreshTimer = setTimeout(async () => {
+      if (!session.user.refreshToken) {
+        return;
+      }
       try {
         const newTokens = await refreshAuthToken(session.user.refreshToken);
-
-        if (newTokens) {
-          await update({
+        if (newTokens && newTokens.accessToken && newTokens.refreshToken) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)